OMCompiler/m4/lapack.m4

```diff
@@ -22,6 +23,7 @@ AC_DEFUN([OMC_AC_LAPACK], [
     done
     LIBS=""
 
+    # Auto detect using pkg-config
     if test "$LD_LAPACK" = "auto"; then
```

Comment:
This indentation might be wrong.

Reply:
It looks correct to me. I wanted to describe the if-case.
Or what exectly do ypu mean?

Reply:
Never-mind, my bad. I was looking at the github collapsed code and it seemed the `if ` at line `8` was closed by the done at `22`. The for loop was hidden by the collapsing.